### DISCUSSION TITLE
Ignore git repo in file picker

### DISCUSSION THREAD

**Discussion by Dimev** [helix-editor/helix#12137] [Q&A]
Is there a way to ignore the git repo when I'm in a directory inside it?
I don't want to browse through all files in the repo if I'm not in it's main directory.

**Comment by kirawi** [comment]
You can create a `.helix` folder to manually set a root

**Reply by Dimev** [reply]
Ah. so there's no way to make this default behaviour?

**Comment by noahsettersten** [comment]
While `Space + f` opens the file picker at the root of the current repo, you can use `Space + F` to open the file picker in the current working directory instead. That might help!
